### PR TITLE
Mesoamerican armor redux

### PR DESCRIPTION
**Pull request by KeremBabaG**
#### Summary
Content "adds Aztec armor"
revival of #71783

#### Purpose of change
Add some Mesoamerican armor because it's COOL DAMMIT

#### Describe the solution
Adds the Ichcahuipilli, a gambeson like Aztec armor that is made from quilted cotton soaked into brine to crystallize and thus make it more protective

Adds the hardwood helmet that I still don't know the name of that is shaped like an eagle or other stuff

#### Describe alternatives you've considered
none, these WILL be added

#### Testing
It's there. Maybe change the warmth and encumberance values a bit.

#### Additional context
[BİTCH IM BACK OUT MY COMA](https://youtu.be/bvBfiRWLj_0?si=9AoEO4axXphop8jQ&t=65)
once again thanks to dgl on discord for the idea

### CODE REVIEW

File: data/json/items/armor/helmets.json [hunk 1]
```diff
@@ -546,6 +546,67 @@
     ],
     "melee_damage": { "bash": 2 }
   },
+  {
+    "id": "helmet_aztec",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "hardwood eagle helmet" },
+    "description": "A hardwood mesoamerican helmet that resembles the head of an eagle.  It was worn by the mesoamerican nobles both ceremonially and for protection.  Inside is lined with cotton; and it is ornamented with many feathers, carvings and colors.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/helmets.json:554]:
```suggestion
    "description": "A hardwood Mesoamerican helmet that resembles the head of an eagle.  It was worn by the Mesoamerican nobles both ceremonially and for protection.  Its inside is lined with cotton, and it is ornamented with many feathers, carvings and colors.",
```

Comment (KeremBabaG) [reply data/json/items/armor/helmets.json:554]: changed in the next commit

**Comment by Kantonine** [comment]
> Adds the hardwood helmet that I still don't know the name of that is shaped like an eagle or other stuff
The Cuacalalatli?

**Comment by KeremBabaG** [comment]
> The Cuacalalatli?

Yeahh that seems to be it

**Pull request merged**